contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,204 @@
+# Encryption
```

Comment:
Secrets are so fundamental to applications, and the diversity of key stores so broad, that we need to be able to do two things:

1. Store secrets and help distribute them to pods, in a basic and secure fashion
2. Enable deep integration with sophisticated key store, management, and audit solutions

So those are the 80% and the 20%.  The 80% has to work - the 20% has to be correct.  It has to be possible to transition from 1 to 2.  The items on the secrets roadmap roughly talk about enabling the remainder of the 80% and the transition to the 20%.